backend/routes/indexer/utils.go

```diff
@@ -0,0 +1,25 @@
+package indexer
+
+import (
+	"fmt"
+	"math/big"
+	"strings"
+)
+
+func combineLowHigh(tokenIdLowHex, tokenIdHighHex string) (*big.Int, error) {
+	// Remove leading zeros from the hex strings
+	tokenIdLowHex = strings.TrimLeft(tokenIdLowHex, "0")
+	tokenIdHighHex = strings.TrimLeft(tokenIdHighHex, "0")
```

Comment:
You wont want to trim all the left 16 0s that are unused.

For example :
High: `00000000000000000000000000000015`
Low:  `00000000000000000000000000001234`
would give `151234`, which is incorrect. It should give `150000000000001234`

Reply:
oh! my bad, I'll fix it now.